core/frontend/src/components/video-manager/VideoManager.vue

```diff
@@ -59,8 +59,11 @@ export default Vue.extend({
       return !this.video_devices.isEmpty()
     },
     video_devices(): Device[] {
-      // Check if a device provides H264
-      function has_h264(device: Device): boolean {
+      function has_supported_encode(device: Device): boolean {
+        if (settings.is_pirate_mode) {
+          return true
+        }
+
         return !device.formats.filter((format: Format) => format.encode === VideoEncodeType.H264).isEmpty()
```

Comment:
YUYV and MJPG will only be available in Pirate Mode for now?

Reply:
yes